package.json

```diff
@@ -22,7 +22,9 @@
   "devDependencies": {
     "eslint": "2.8.0",
     "gulp": "gulpjs/gulp#73eced0",
-    "skatejs-build": "skatejs/build#22dc0e2",
+    "incremental-dom": "^0.3.0",
+    "skatejs-build": "skatejs/build#e25d583fea5f3fc590aed2ee274d37daebf279a2",
+    "skatejs-named-slots": "skatejs/named-slots#00daacd",
```

Comment:
For a 1.0 it would be cool to also have a named-slots release instead of a sha I think